osgp/platform/osgp-adapter-ws-core/src/main/java/org/opensmartgridplatform/adapter/ws/core/application/mapping/UpdatedDeviceConverter.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.ws.core.application.mapping;
+
+import org.opensmartgridplatform.adapter.ws.schema.core.devicemanagement.BaseDeviceModel;
+import org.opensmartgridplatform.adapter.ws.schema.core.devicemanagement.Device;
+import org.opensmartgridplatform.adapter.ws.schema.core.devicemanagement.UpdatedDevice;
+import org.opensmartgridplatform.domain.core.entities.DeviceModel;
+import org.opensmartgridplatform.domain.core.entities.Ssld;
+
+import ma.glasnost.orika.MappingContext;
+import ma.glasnost.orika.converter.BidirectionalConverter;
+import ma.glasnost.orika.metadata.Type;
+
+public class UpdatedDeviceConverter extends BidirectionalConverter<UpdatedDevice, Ssld> {
+
+    @Override
+    public Ssld convertTo(final UpdatedDevice source, final Type<Ssld> destinationType,
+            final MappingContext mappingContext) {
+        final Device device = this.mapperFacade.map(source, Device.class);
+        final Ssld ssld = this.mapperFacade.map(device, Ssld.class);
+        // Just to be sure
+        ssld.setDeviceModel(this.mapperFacade.map(source.getDeviceModel(), DeviceModel.class));
+        return ssld;
+    }
+
+    @Override
+    public UpdatedDevice convertFrom(final Ssld source, final Type<UpdatedDevice> destinationType,
+            final MappingContext mappingContext) {
+        final Device device = this.mapperFacade.map(source, Device.class);
+        final UpdatedDevice updatedDevice = this.mapperFacade.map(device, UpdatedDevice.class);
+        // Just to be sure;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)